boards/arm64/intel_socfpga_agilex5_socdk/intel_socfpga_agilex5_socdk_defconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# The Zephyr build from this defconfig is execpted to boot from
+#Intel Arm Trusted Firmware (ATF).
```

Comment:
What is the `Intel Arm Trusted Firmware`, it's an Intel fork of ATF or ?

Reply:
Yes, it is - where we have support for Intel SoC FPGA product lines.
And we are using the ATF as a boot loader to Zephyr.